lms/djangoapps/shoppingcart/tests/test_models.py

```diff
@@ -56,7 +56,13 @@ def setUp(self):
         self.course_key = course.id
         self.other_course_keys = []
         for __ in xrange(1, 5):
-            self.other_course_keys.append(CourseFactory.create().id)
+            course_key = CourseFactory.create().id
+            CourseMode.objects.create(
+                course_id=course_key,
+                mode_slug=CourseMode.HONOR,
```

Comment:
Yes -- since all these tests involve honor certificates, the test course needs an honor mode. Previously this existed by default; now it needs to be added explicitly.